upgrade/sql/9.0.0.sql

```diff
@@ -13,3 +13,7 @@ ALTER TABLE `PREFIX_feature_flag` ADD `type` VARCHAR(64) DEFAULT 'env,dotenv,db'
 
 /* Increase size of customized data - https://github.com/PrestaShop/PrestaShop/pull/31109 */
 ALTER TABLE `PREFIX_customized_data` MODIFY `value` varchar(1024) NOT NULL;
+
+DELETE FROM `PREFIX_cart` WHERE `id_address_delivery` > 0 && id_address_delivery` NOT IN (SELECT `id_address` FROM `PREFIX_address`);
```

Comment:
```suggestion
DELETE FROM `PREFIX_cart` WHERE `id_address_delivery` > 0 && `id_address_delivery` NOT IN (SELECT `id_address` FROM `PREFIX_address`);
```

I think you're missing a backquote here.

Reply:
@tleon yea, sorry. Butter fingers :))